homeassistant/components/fastdotcom/manifest.json

```diff
@@ -1,11 +1,11 @@
 {
   "domain": "fastdotcom",
   "name": "Fast.com",
-  "codeowners": ["@rohankapoorcom", "@erwindouna"],
+  "codeowners": ["@rohankapoorcom", "@erwindouna", "@rosenrot"],
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/fastdotcom",
   "iot_class": "cloud_polling",
   "loggers": ["fastdotcom"],
-  "requirements": ["fastdotcom==0.0.3"],
+  "requirements": ["fastdotcom==0.0.6"],
```

Comment:
lets bump the library in a separate PR as noted in point 7 of https://developers.home-assistant.io/docs/review-process/#creating-the-perfect-pr

since `fast_com()` output has changed, you can make it so it extracts the download speed on the coordinator for the version bump.